code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -381,6 +381,16 @@
 /obj/item/ammo_magazine/m545saw/empty
 	initial_ammo = 0
 
+/obj/item/ammo_magazine/m545m116 //spades why did you change 5.56 to 5.45 you make no sense
```

Comment:
Because we already have a space AK and the fact that it used 5.56 ammo TRIGGERS ME